selene-lib/src/lints/roblox_incorrect_roact_usage.rs

```diff
@@ -136,6 +142,8 @@ struct InvalidEvent {
 struct InvalidProperty {
     class_name: String,
     property_name: String,
+    property_value: Box<ast::Expression>,

```

Comment:
Why does this need to be boxed? If it's just because we're taking it from something that's boxed, deref first.